config/database.yml

```diff
@@ -1,25 +1,23 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
-default: &default
-  adapter: sqlite3
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
+mysql: &mysql
+  adapter: mysql2
+  encoding: utf8
+  host: <%= ENV.fetch('DB_HOST') %>
+  port: 3306
+  username: <%= ENV.fetch('DB_USER') %>
+  password: <%= ENV.fetch('DB_PASS') %>
+
+test:
+  <<: *mysql
+  database: govwifi_admin_test
 
 development:
-  <<: *default
-  database: db/development.sqlite3
+  <<: *mysql
+  database: development
 
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
-test:
-  <<: *default
-  database: db/test.sqlite3
+staging:
+  <<: *mysql
+  database: govwifi_admin_staging
```

Comment:
We need to use env variables here no?

Reply:
Why? Will we have many different database names per environment?

Reply:
Sure ok yup